### DISCUSSION TITLE
Why have not CLI for ubuntu ?

### DISCUSSION THREAD

**Discussion by dungla2011** [Chia-Network/chia-blockchain#5304] [Support]
I've not found CLI for ubuntu  here:
https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference
Where can I found chia executable?
Tks!

**Comment by lvcivs** [comment]
Depends on how you installed. Installation from github: the executable is in the folder `chia-blockchain`. Installation from the .deb package: it is in  ` /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/`

**Reply by dungla2011** [reply]
I find, but it seem not found:

![image](https://user-images.githubusercontent.com/7878963/118572107-d1136f80-b7a9-11eb-9922-7b474a65a01b.png)

**Comment by dungla2011** [comment]
oh, It show well after activate!
Thank you!

![image](https://user-images.githubusercontent.com/7878963/118572665-2e5bf080-b7ab-11eb-9bc4-12c7d53f1d78.png)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
